bugbot/rules/triage_owner_rotations.py

```diff
@@ -95,10 +96,25 @@ def get_email_data(self, date: str) -> List[dict]:
                 ),
                 "new_triage_owner": new_triager.bugzilla_email,
                 "has_put_error": new_triager.component in failures,
+                "cc_emails": [
+                    self.component_triagers.get_current_triage_owner(
+                        new_triager.component
+                    ),
+                    new_triager.bugzilla_email,
+                ],
```

Comment:
We do not need to pass `cc_emails` here. We could simply use `old_triage_owner` and `new_triage_owner` in the `get_cc_emails()` method.